### PR TITLE
#1628 Refactor sort table component making it more reusable

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1628 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Nodes/Nodes.js [hunk 1]
```diff
@@ -67,24 +65,16 @@ const Nodes = props => {
...
   const rows = useNodes.map(wk => {
-    return createData(
-      wk.name,
-      wk.port,
-      moment.unix(wk.lastModified / 1000).format('YYYY-MM-DD HH:mm:ss'),
-    );
+    return {
+      name: wk.name,
+      port: wk.port,
+      lastModified: moment
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Nodes/Nodes.js]: Could you use the `getDateFromTimestamp` here?

**Comment by konekoya** [comment]
@wu87988622 Please don't merge this PR until the release branch is created by @jackyoh

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Nodes/Nodes.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Nodes/Nodes.js]: ok

**Comment by wu87988622** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager

**Pull request closed without merging**